Makefile

```
@@ -1,24 +1,111 @@
-.PHONY: all build run run-react run-laravel clean test
+.PHONY: help build test clean install dev lint fmt deps release docker
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)